content/en/docs/concepts/services-networking/service.md

```diff
@@ -1398,17 +1398,7 @@ in-memory locking). Kubernetes also uses controllers to check for invalid
 assignments (eg due to administrator intervention) and for cleaning up allocated
 IP addresses that are no longer used by any Services.
 
-#### IP address ranges for `type: ClusterIP` Services {#service-ip-static-sub-range}
-
-{{< feature-state for_k8s_version="v1.25" state="beta" >}}
-However, there is a problem with this `ClusterIP` allocation strategy, because a user
-can also [choose their own address for the service](#choosing-your-own-ip-address).
-This could result in a conflict if the internal allocator selects the same IP address
-for another Service.
-
-The `ServiceIPStaticSubrange`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default in v1.25
-and later, using an allocation strategy that divides the `ClusterIP` range into two bands, based on
+Services are using an allocation strategy that divides the `ClusterIP` range into two bands, based on
 the size of the configured `service-cluster-ip-range` by using the following formula
 `min(max(16, cidrSize / 16), 256)`, described as _never less than 16 or more than 256,
 with a graduated step function between them_. Dynamic IP allocations will be preferentially
```

Comment:
```suggestion
`min(max(16, cidrSize / 16), 256)`. To paraphrase that formula: _the allocation bands are never
smaller than a range with 16 IP addresses, and neither allocation band is ever more than 256
addresses_; between those limits Kubernetes applies a graduated step function to determine the
size of each allocation band.
Dynamic IP allocations will be preferentially
```
How does this work for IPv6? Is it still a maximum band of 256 IP addresses (a /120 range)?

Reply:
yep, the allocator doesn't depend on the ip family, only on the size of the range allocated